detection/evasion/unexpected-var-run-macos.sql

```diff
@@ -52,6 +52,7 @@ WHERE
     'hdiejectd.pid',
     'installd.commit.pid',
     'kdc.pid',
+    'MobileAssetCritialDomainsUpdated.plist',
```

Comment:
This was an official addition in iOS 18.2 -- https://github.com/blacktop/ipsw-diffs/blob/2ec332e3f486922bf698cb2089adb2ec543cee67/18_2_22C5125e__vs_18_2_22C5131e/Entitlements.md?plain=1#L763

That said, I still reached out to Apple about why it's spelled incorrectly.